subprojects/docs/src/docs/userguide/gradleWrapper.adoc

```diff
@@ -136,24 +232,22 @@ Embedding credentials in the `distributionUrl` in the `gradle/wrapper/gradle-wra
 ====
 
 [source,properties]
-.`gradle-wrapper.properties`
+.`gradle/wrapper/gradle-wrapper.properties`
 ----
-
- distributionUrl=https://username:password@somehost/path/to/gradle-distribution.zip
-
+distributionUrl=https://username:password@somehost/path/to/gradle-distribution.zip
 ----
 ====
 
 This can be used in conjunction with a proxy, authenticated or not. See <<sec:accessing_the_web_via_a_proxy>> for more information on how to configure the `Wrapper` to use a proxy.
 
 [[sec:verification]]
-=== Verification of downloaded Gradle distributions
+==== Verification of downloaded Gradle distributions
 
 The Gradle Wrapper allows for verification of the downloaded Gradle distribution via SHA-256 hash sum comparison. This increases security against targeted attacks by preventing a man-in-the-middle attacker from tampering with the downloaded Gradle distribution.
 
 To enable this feature, download the `.sha256` file associated with the Gradle distribution you want to verify.
 
-==== Downloading the SHA-256 file
+===== Downloading the SHA-256 file
 
 You can download the `.sha256` file by clicking on one of the `sha256` links on whichever page you used to download your distribution:
```

Comment:
Perhaps it would be better to just link to http://services.gradle.org/distributions/ and http://services.gradle.org/distributions-snapshots/ for this paragraph. These links are soon going to be non-existent.

Reply:
Sure, if that's the plan then let's do it.